src/curated-corpus/pages/SchedulePage/SchedulePage.tsx

```diff
@@ -557,13 +564,7 @@ export const SchedulePage: React.FC = (): ReactElement => {
         formikHelpers.setSubmitting(false);
 
         // Hide the Schedule Item Form modal
-        toggleScheduleItemModal();
-
-        // Unset the corpus item that was added manually so that
-        // the ScheduleItemModal specific to manually added stories
-        // (with reasons for scheduling for the New Tab (US) surface)
-        // does not show up when a reorder/reschedule action is triggered.
-        setApprovedItem(undefined);
+        toggleManualScheduleItemModal();
```

Comment:
Unsetting the last worked on approved item is no longer needed as we're not using the same on/off toggle for the Schedule Item modal any more, so this can be taken out. It was a half-way fix, but it would not have worked consistently all the time.